scheduler/actions/images/encoding/codec.go

```diff
@@ -82,7 +83,8 @@ func NewImageCodec(fileExt string) ImageCodec {
 
 // Decode reads an image from the provided reader
 func (c defaultCodec) Decode(reader io.Reader) (image.Image, error) {
-	return imaging.Decode(reader)
```

Comment:
Sorry to bother again :-) 
Does this have any perf implication for formats that do not support exif ? It seems that exif is JPG/TIFF only, maybe we should use the "standard" decode for BMP, PNG, WEBP...?

Reply:
There might be a minor performance hit. For our case I believe is better to configure it upon decoder instantiation, to avoid having non-intuitive rotation only for one format type. Something like:
```go
	t.codec = encoding.NewImageCodec(fileFormat, &encoding.CodecOptions{
		EnforceExifOrientation: true,
	})
```

Then in case is not enforced it uses the standard encoding